konnect-runtime-setup.sh

```diff
@@ -214,7 +230,7 @@ login() {
 get_control_plane() {
     log_debug "=> entering control plane metadata retrieval phase"
 
-    ARGS="--cookie ./$KONNECT_HTTP_SESSION_NAME -X GET --url $KONNECT_API_URL/api/runtime_groups/$KONNECT_CONTROL_PLANE"
+    ARGS="--cookie ./$KONNECT_HTTP_SESSION_NAME -X GET --url $HTTP_PROTOCOL://$KONNECT_REGION.$KONNECT_API_HOST/api/runtime_groups/$KONNECT_CONTROL_PLANE"
```

Comment:
Shouldn't this be `$HTTP_PROTOCOL://$KONNECT_REGION.$KONNECT_API_HOST/konnect-api/api/runtime_groups/$KONNECT_CONTROL_PLANE`, or what do you expect `$KONNECT_API_HOST` to be?

Reply:
yup, this is correct. Updated the script to add `/konnect-api` after the host.